api/column_family.cc

```diff
@@ -79,14 +79,14 @@ future<json::json_return_type>  get_cf_stats(http_context& ctx,
 }
 
```

Comment:
table.cc need merge?

Reply:
I was referring to the line in the patch changelog that says "table.cc need merge"

Reply:
I figure it out eventualy when going over the commit messages